artifacts_test.py

```diff
@@ -438,6 +438,11 @@ def test_scylla_service(self):
                 self.verify_docker_latest_match_release()
 
     def run_scylla_doctor(self):
+        # Scylla Docker does not collect Scylla cluster logs - https://github.com/scylladb/field-engineering/issues/2288
+        if self.node.parent_cluster.cluster_backend == "docker":
```

Comment:
didn't you said you'll just skip the logs checking ?

Reply:
I found that there are failed collectors due to missed utility.
I asked them about running SD in docker and skip this test meanwhile:
https://github.com/scylladb/field-engineering/issues/2288

Reply:
part of those we should address, like installing `ethtool`
others are cause it's docker, and SD would need to address